admin/load.php

```diff
@@ -210,6 +210,9 @@ function perflab_get_focus_areas() {
 		'images'       => array(
 			'name' => __( 'Images', 'performance-lab' ),
 		),
+		'javascript'   => array(
```

Comment:
Maybe this is not required. Should we include the `partytown` module in the `js-and-css` focus area?

Reply:
Yeah, let's include as part of `js-and-css`

Reply:
Done in https://github.com/WordPress/performance/pull/556/commits/335daaa1ace427486f7b2023963ffeda72f6befc